src/tasks/daily.ts

```diff
@@ -164,6 +166,53 @@ function voterSetup(): void {
   visitUrl(`choice.php?option=1&whichchoice=1331&g=${monsterVote}&local[]=${init}&local[]=${init}`);
 }
 
+function chibiSetup(): void {
+  if (!have($item`ChibiBuddy™ (on)`) && !have($item`ChibiBuddy™ (off)`)) return;
+  // We need to name our buddy when we turn it on!
+  const chibiNames = [
+    "SSBBHax",
+    "Frasier Crane",
+    "William",
+    "Gamuel Sauce",
+    "Axis Shadowbaenimus",
+    "Dale Cooper",
+  ];
+  // It is possible our Buddy died after rollover, and will turn to (off) after using it
+  // It is also possible we ascended with a Buddy(on) in which case it will reset immediately without the choice adventure on use
+  if (have($item`ChibiBuddy™ (on)`)) {
+    directlyUse($item`ChibiBuddy™ (on)`);
+    if (handlingChoice()) {
+      // This is the choice option for chibi chat
+      if (availableChoiceOptions()[5]) {
```

Comment:
this may(?) read nicer with the `in` operator. That or you can check `Object.values(availableChoiceOptions()).includes(` if that reads nicer

Reply:
I personally liked the way this read, we're seeing the choice options, and whether 5 exists, so I feel it's both succinct and clear?

I could certainly be wrong about that though